runtime/gc_glue_java/ScavengerDelegate.hpp

```diff
@@ -179,6 +178,8 @@ class MM_ScavengerDelegate : public MM_BaseNonVirtual {
 	bool initialize(MM_EnvironmentBase *env);
 	void tearDown(MM_EnvironmentBase *env);
 
+	bool doContinuationObject(MM_EnvironmentStandard *env, omrobjectptr_t objectPtr, MM_ScavengeScanReason reason);
```

Comment:
move it up above init/teardown, perhaps close to setShouldScavengeContinuationObjects?

Reply:
BTW, we are somewhat inconsistent in naming. For example, MarkingDelegate calls this 'scan', not 'do'.... Not sure what is the right one, and if I want to fix it now (this is already growing too much).